cmd/conf/database.go

```diff
@@ -16,18 +16,21 @@ type PersistentConfig struct {
 	GlobalConfig string `koanf:"global-config"`
 	Chain        string `koanf:"chain"`
 	Handles      int    `koanf:"handles"`
+	Ancients     string `koanf:"ancients"`
 }
 
 var PersistentConfigDefault = PersistentConfig{
 	GlobalConfig: ".arbitrum",
 	Chain:        "",
 	Handles:      512,
+	Ancients:     "",
 }
 
 func PersistentConfigAddOptions(prefix string, f *flag.FlagSet) {
 	f.String(prefix+".global-config", PersistentConfigDefault.GlobalConfig, "directory to store global config")
 	f.String(prefix+".chain", PersistentConfigDefault.Chain, "directory to store chain state")
 	f.Int(prefix+".handles", PersistentConfigDefault.Handles, "number of file descriptor handles to use for the database")
+	f.String(prefix+".ancients", PersistentConfigDefault.Ancients, "directory of ancients where the chain freezer can be opened")
```

Comment:
I've validated this works locally :) One small nit though, geth refers to it as the "ancient database" not the "ancients database". Could we rename this option from ancients to ancient to match it?

Reply:
done.